packages/app-mobile/components/ScreenHeader/index.tsx

```diff
@@ -48,8 +48,6 @@ interface ScreenHeaderProps {
 	selectedFolderId: string;
 	notesParentType: string;
 	noteSelectionEnabled: boolean;
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-	parentComponent: any;
```

Comment:
`SearchScreen` is now a function component. As such, it can't provide a `parentComponent` prop. However, the `parentComponent` prop is used only to show a confirmation dialog. Replacing this with `shim.showConfirmationDialog` allows the `parentComponent` prop to be removed.